index.html

```diff
@@ -184,5 +184,38 @@ <h2 class="playfairFont margTop1">About</h2>
       </div>
     </section>
   </main>
+
+  <section class="artDeco2">
+    <h2 class="playfairFont">Get Started</h2>
+
+    <p class="loremStyle">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+
+    <form action="https://formspree.io/f/xknavqkz" method="post">
+
+      <label for="name" class="label1">Your name</label>
+      <input type="text" id="name" class="fullWidth2" maxlength="30" placeholder="Full name" required>
+      <label for="email" class="label1">Your email</label>
+      <input type="email" id="email" class="fullWidth2" placeholder="Email address" required>
+      <label for="textarea" class="label1">Your message</label>
+      <textarea id="textarea" class="fullWidth3" maxlength="500" placeholder="Enter your message here..." required></textarea>
```

Comment:
- [ ] As per design, you've done a very great job what is left is just adding left padding to the first two input fields, and likewise a top and left padding to the textarea input fields respectively.
- And likewise remove the resize button at the bottom right corner.

- Your design
<img width="292" alt="image" src="https://user-images.githubusercontent.com/41163571/228553461-91022339-fbae-4740-95fc-5416284706a8.png">

- Figma design
<img width="168" alt="image" src="https://user-images.githubusercontent.com/41163571/228553760-930a4fbb-7f2e-4851-bf46-7c87dd4aba50.png">